src/main/java/com/depaul/cdm/se452/group6/movie/controller/IndexController.java

```diff
@@ -0,0 +1,47 @@
+package com.depaul.cdm.se452.group6.movie.controller;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.ModelAttribute;
+import org.springframework.web.bind.annotation.PostMapping;
+
+import com.depaul.cdm.se452.group6.movie.entity.UserLogin;
+import com.depaul.cdm.se452.group6.movie.service.UserLoginService;
+
+@Controller
+public class IndexController {
+	private UserLoginService userLoginService;
+	
+	public IndexController(UserLoginService userLoginService) {
+		this.userLoginService = userLoginService;
+	}
+	
+	@GetMapping("/")
+    public String root(Model model) {
+		userLoginService.loginSuccess(1L, "admin", "1234");
```

Comment:
I was supposed to comment out this line. Otherwise it will create a login info every time user logs in instead of only once.

Reply:
you can leave this line as is and change userloginservice to model the logservice. it checks the log level and will only log entries to mongo if the application.properties file sets the log level to DEBUG instead of its current value of INFO